pkg/vm/cli/cli_test.go

```diff
@@ -104,7 +104,8 @@ func (e *executor) checkNextLine(t *testing.T, expected string) {
 func (e *executor) checkError(t *testing.T, expectedErr error) {
 	line, err := e.out.ReadString('\n')
 	require.NoError(t, err)
-	require.True(t, strings.HasPrefix(line, "Error: "+expectedErr.Error()))
+	expected := "Error: " + expectedErr.Error()
+	require.True(t, strings.HasPrefix(line, expected), fmt.Errorf("expected `%s`, got `%s`", expected, line))
```

Comment:
Not a path/filepath thing, but somehow a part of 5a43ffedfe1d6c0caa8cae0366d86226e0997698 (which is more of `*:` rather than `cli:` now).